.github/workflows/build.yml

```diff
@@ -145,7 +189,7 @@ jobs:
         with:
           current-version: "${{ env.CURRENT_SEMVER }}"
           # TODO: needs to be configurable https://github.com/christian-draeger/increment-semantic-version#input--output-examples
-          version-fragment: 'bug'
+          version-fragment: "bug"
```

Comment:
Do we need this change ?

Reply:
Is there any specific reason why these quotes are single while the rest of them are double?

Reply:
https://github.com/christian-draeger/increment-semantic-version#example-usage

Reply:
This is fine - I was just curious.